<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FuseManagerUI</class>
 <widget class="QWidget" name="FuseManagerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>352</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbDomainSpecific">
     <property name="whatsThis">
      <string/>
     </property>
     <property name="title">
      <string>Available FUSE services</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="bDelete">
        <property name="text">
         <string>D&amp;elete</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="bAdd">
        <property name="text">
         <string>&amp;Add...</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>81</width>
          <height>47</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="bChange">
        <property name="text">
         <string>Chan&amp;ge...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="bDeleteAll">
        <property name="text">
         <string>Delete A&amp;ll</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" rowspan="5">
       <widget class="QTreeWidget" name="serviceTreeWidget">
        <property name="whatsThis">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;List of FUSE mount services.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
        <property name="rootIsDecorated">
         <bool>false</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <property name="allColumnsShowFocus">
         <bool>true</bool>
        </property>
        <column>
         <property name="text">
          <string>Protocol</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Handler</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
